debezium-connector-mongodb/pom.xml

```diff
@@ -204,7 +204,7 @@
                         <!-- Make these available to the tests via system properties -->
                         <connector.mongodb.hosts>${mongo.data.replicaset.name}/${docker.host.address}:27017</connector.mongodb.hosts>
                         <connector.mongodb.members.auto.discover>false</connector.mongodb.members.auto.discover>
-                        <connector.mongodb.name>mongo1</connector.mongodb.name>
+                        <connector.database.name>mongo1</connector.database.name>
```

Comment:
@jpechane, curious about your thoughts on this one; agreed it's inconsistent with the other connectors, but this goes further than just that single property, it also applies to things like user name etc. So I'm inclined to leave things for MongoDB as they are for the time being and revisit all the property names of this connector at a separate time. WDYT?

If we actually agree on changing it, then I'm wondering btw. whether it shouldn't be "server" here, instead of "database". That's actually another reason why I'm reluctant to changing it, as the scope of this name is a replica set rather than a server in the case of MongoDB.

Reply:
I'd keep the names for mongodb as is now and later on either unify all or nothing to make sure the naming is consitent.

Reply:
Right, that's my thinking as well. So @krizhan, could you reverte the part of the change that pertains to MongoDB? I think then we're good to go. Thanks!

Reply:
No worries and sounds good. Btw in that case, I would have to move SERVER_NAME to RelationalDatabaseConnectorConfig rather than CommonConfig otherwise it would be a required field for MongoDB

Reply:
> I would have to move SERVER_NAME to RelationalDatabaseConnectorConfig

+1